# German translations of dnf-plugins-extras.
# Mario Blättermann <mario.blaettermann@gmail.com>, 2015.
# Mario Blättermann <mario.blaettermann@gmail.com>, 2015. #zanata
# Mario Blättermann <mario.blaettermann@gmail.com>, 2016. #zanata
# Marek Blaha <mblaha@redhat.com>, 2018. #zanata
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-06-05 10:59+0200\n"
"PO-Revision-Date: 2018-04-06 02:05+0000\n"
"Last-Translator: Marek Blaha <mblaha@redhat.com>\n"
"Language-Team: German <debian-l10n-german@lists.debian.org>\n"
"Language: de\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"X-Generator: Zanata 4.6.2\n"

#. Translators: This string is only used in unit tests.
#: ../plugins/system_upgrade.py:43
msgid "the color of the sky"
msgstr ""

#: ../plugins/system_upgrade.py:59
msgid "Need a --releasever greater than the current system version."
msgstr ""

#. Translators: do not change "reboot" here
#: ../plugins/system_upgrade.py:61
msgid ""
"Download complete! Use 'dnf system-upgrade reboot' to start the upgrade.\n"
"To remove cached metadata and transaction use 'dnf system-upgrade clean'"
msgstr ""

#: ../plugins/system_upgrade.py:64
msgid "Sorry, you need to use 'download --releasever' instead of '--network'"
msgstr ""

#: ../plugins/system_upgrade.py:103
#, python-format
msgid "Screen blanking can't be disabled: %s"
msgstr ""

#: ../plugins/system_upgrade.py:124
#, python-format
msgid "Failed loading state file: %s, continuing with empty state."
msgstr ""

#: ../plugins/system_upgrade.py:253
msgid "The following boots appear to contain upgrade logs:"
msgstr ""

#: ../plugins/system_upgrade.py:263
msgid "-- no logs were found --"
msgstr ""

#: ../plugins/system_upgrade.py:278
msgid "Cannot find logs with this index."
msgstr ""

#: ../plugins/system_upgrade.py:287
msgid "Unable to match systemd journal entry"
msgstr ""

#: ../plugins/system_upgrade.py:306
msgid "Prepare system for upgrade to a new release"
msgstr ""

#: ../plugins/system_upgrade.py:316
msgid "keep installed packages if the new release's version is older"
msgstr ""

#: ../plugins/system_upgrade.py:320
msgid "which logs to show"
msgstr ""

#: ../plugins/system_upgrade.py:376
msgid ""
"Before you continue ensure that your system is fully upgraded by running "
"\"dnf --refresh upgrade\". Do you want to continue"
msgstr ""
"Bevor Sie fortfahren, stelle Sie bitte sicher, das alle Aktualisierungen "
"installiert sind. Dies kann durch den Aufruf von \"dnf --refresh upgrade\" "
"erfolgen. Wollen Sie fortfahren?"

#: ../plugins/system_upgrade.py:380
msgid "Operation aborted."
msgstr ""

#: ../plugins/system_upgrade.py:440
msgid "system is not ready for upgrade"
msgstr ""

#: ../plugins/system_upgrade.py:442
msgid "upgrade is already scheduled"
msgstr ""

#: ../plugins/system_upgrade.py:448
msgid "trigger file does not exist. exiting quietly."
msgstr ""

#: ../plugins/system_upgrade.py:451
msgid "another upgrade tool is running. exiting quietly."
msgstr ""

#. Translators: do not change "reboot" here
#: ../plugins/system_upgrade.py:457
msgid "use 'dnf system-upgrade reboot' to begin the upgrade"
msgstr ""

#: ../plugins/system_upgrade.py:474
msgid "Rebooting to perform upgrade."
msgstr ""

#: ../plugins/system_upgrade.py:496 ../plugins/system_upgrade.py:502
msgid "Starting system upgrade. This will take a while."
msgstr ""

#: ../plugins/system_upgrade.py:528
#, python-format
msgid "Unable to match package: %s"
msgstr ""

#: ../plugins/system_upgrade.py:533
msgid "Unable to match some of packages"
msgstr ""

#: ../plugins/system_upgrade.py:536
msgid "Cleaning up downloaded data..."
msgstr ""

#: ../plugins/system_upgrade.py:580
msgid "Download finished."
msgstr ""

#: ../plugins/system_upgrade.py:584 ../plugins/system_upgrade.py:585
msgid "Upgrade complete! Cleaning up and rebooting..."
msgstr ""

#: ../plugins/tracer.py:70
msgid "For more information run:"
msgstr "Weitere Informationen erhalten Sie mit:"

#: ../plugins/tracer.py:97
msgid "Call to Tracer API ended unexpectedly:"
msgstr ""

#: ../plugins/tracer.py:99
msgid ""
"Please visit https://github.com/FrostyX/tracer/issues and submit the issue. "
"Thank you"
msgstr ""

#: ../plugins/tracer.py:101
msgid "We apologize for any inconvenience"
msgstr "Wir bitten, die Unannehmlichkeiten zu entschuldigen"

#: ../plugins/rpm_conf.py:65
msgid "Adding '{}' to list of handling packages for rpmconf"
msgstr "»{}« wird zur Liste der Pakete für rpmconf hinzugefügt"

#: ../plugins/rpm_conf.py:71
msgid "rpmconf plugin will not run in non-interactive mode"
msgstr ""

#: ../plugins/rpm_conf.py:84
msgid "ignoring sys.exit from rpmconf due to missing MERGE variable"
msgstr ""

#: ../plugins/rpm_conf.py:88
msgid "ignoring sys.exit from rpmconf due to missing file"
msgstr ""

#: ../plugins/snapper.py:49
#, python-format
msgid "connect to snapperd failed: %s"
msgstr "Verbindung zu snapperd fehlgeschlagen: %s"

#: ../plugins/snapper.py:55
msgid "creating pre_snapshot"
msgstr ""

#: ../plugins/snapper.py:61
#, python-format
msgid "created pre_snapshot %d"
msgstr ""

#: ../plugins/snapper.py:65
#, python-format
msgid "creating pre_snapshot failed: %s"
msgstr ""

#: ../plugins/snapper.py:74
msgid "skipping post_snapshot because creation of pre_snapshot failed"
msgstr ""

#: ../plugins/snapper.py:80
msgid "creating post_snapshot"
msgstr ""

#: ../plugins/snapper.py:85
#, python-format
msgid "created post_snapshot %d"
msgstr ""

#: ../plugins/snapper.py:89
#, python-format
msgid "creating post_snapshot failed: %s"
msgstr ""

#: ../plugins/kickstart.py:59
msgid "Install packages defined in a kickstart file on your system"
msgstr ""
"In einer Kickstart-Datei auf Ihrem System definierte Pakete installieren"

#: ../plugins/kickstart.py:64
msgid "kickstart file"
msgstr ""

#: ../plugins/kickstart.py:82
#, python-format
msgid "file cannot be parsed: %s"
msgstr "Datei kann nicht eingelesen werden: %s"

#: ../plugins/kickstart.py:98
#, python-format
msgid "No package %s available."
msgstr "Kein Paket %s verfügbar."

#: ../plugins/kickstart.py:103
msgid "Nothing to do."
msgstr "Nichts zu tun."

#: ../plugins/torproxy.py:31
msgid "Disabling torproxy plugin: cannot connect to the Tor network"
msgstr ""
